testutil/testutil.go

```diff
@@ -19,25 +19,27 @@ var protocPath = func() string {
 	return "protoc"
 }
 
+// FileDescriptorSpec defines a specification for generating a FileDescriptorProto
 type FileDescriptorSpec struct {
+	// Filename is the output of the returned FileDescriptorProto.GetName().
 	Filename string
+	// Template defines a text/template to use for the proto source.
 	Template string
-	Data     interface{}
-	Deps     []*descriptorpb.FileDescriptorProto
+	// Data is plugged into the template to create the full source code.
+	Data interface{}
+	// Deps are any additional FileDescriptorProtos that the protocol compiler will need for the source.
+	Deps []*descriptorpb.FileDescriptorProto
+	// AdditionalProtoPaths are any additional proto_paths that the protocol compiler will need for the source.
+	AdditionalProtoPaths []string
 }
 
-// MustCreateFileDescriptorProtoFromTemplate creates a *descriptorpb.FileDescriptorProto from a string template and data.
-func MustCreateFileDescriptorProtoFromTemplate(spec FileDescriptorSpec) *descriptorpb.FileDescriptorProto {
-	tmpl := template.Must(template.New("test").Parse(spec.Template))
-	b := new(bytes.Buffer)
-	if err := tmpl.Execute(b, spec.Data); err != nil {
+// MustCreateFileDescriptorProtoFromSpec creates a *descriptorpb.FileDescriptorProto from a string template and data.
+func MustCreateFileDescriptorProtoFromSpec(spec FileDescriptorSpec) *descriptorpb.FileDescriptorProto {
```

Comment:
I think we can shorten the function name by removing `FromSpec`.

Reply:
Done.